src/main/java/redis/clients/jedis/util/RedisVersion.java

```diff
@@ -0,0 +1,41 @@
+package redis.clients.jedis.util;
+
+import java.util.Arrays;
+
+public class RedisVersion implements Comparable<RedisVersion> {
```

Comment:
> we can even package private this class in `redis.clients.jedis` package.

WDYT?

Reply:
it is a change of the package of class and unit tests of it (if we dont want to do some fancy code). My experience is that these decisions could easily end up with a crowded and unrelated stuff get into to the same bag, especially with the big projects. I would like to avoid moving things around if i am not %100 sure, like this one. Are u sure on moving it ? 
sorry that it is like i reversed the question somehow to you =B

Reply:
> My experience is that these decisions could easily end up with a crowded and unrelated stuff get into to the same bag, especially with the big projects.

Agree. To handle that my approach is to avoid adding anything `public` as much as we can.
My experience is also that users find weird ways of using `public` stuff. In some rare cases, we end up unwillingly continuing to support those. That's my another reason for _not-public_.

So,

> Are u sure on moving it ?

Yes.

Reply:
done, moved it to redis.clients.jedis.csc.
still not feeling much convinced on moving things in hierarchy in general to just to make them hidden, anyway i don't take this as a big deal.